cmd/veneur/main.go

```diff
@@ -71,6 +71,16 @@ func main() {
 		}
 	}()
 
+	// Accept connections on the TCP port if enabled
```

Comment:
I think you'll want this in a separate goroutine (similar to the trace reader), so it can start asynchronously.